components/schema-migrator/migrations/director/20240119172919_add-formation-assignments-timestamps.up.sql

```diff
@@ -0,0 +1,21 @@
+BEGIN;
+
+ALTER table formations
+    ADD COLUMN last_state_change_timestamp TIMESTAMP,
+    ADD COLUMN last_notification_sent_timestamp TIMESTAMP;
+
+UPDATE formations SET last_state_change_timestamp = now()
```

Comment:
You can use DEFAULT value during column creation instead of updating each row of the table.